### PR TITLE
Initial Koa release notes

### PR DESCRIPTION
**Pull request by nedbat**
Copied from
https://docs.google.com/document/d/14DZIIIBIO_pOjPX9jjIqwG2pjUM2yr26S_aLmvA06qU/edit# (private to edX)

You can see them as they will be published at https://edx.readthedocs.io/projects/open-edx-release-notes/en/nedbat-koa-release-notes/koa.html

## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by nedbat** [comment]
@regisb Let's review what we have now.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 1]
```diff
@@ -4,7 +4,202 @@
...
+Content Search Tools
+....................
+
+**Course Content Search Updates**: Visual updates to the course content search results page now clarifies the result location and type for learners.
+
+
+Mobile Application Experiences
+------------------------------
+
+**Mobile Log In & Registration**: A few updates to the iOS and Android registration experiences are important to call out with the aim of improving ease of use. Instead of scrolling and selecting a country now a user can write into the spinner view and this will filter accordingly. Input fields are now validated once a user's focus changes to the next field (previously the validation used to happen on the fly as the user typed into a field). A new API is now used to validate the registration prior to submission. (Please see details in the full API list here.)
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst]: > (Please see details in the full API list here.)

Is there a missing link to the API list at the end of this paragraph?

File: en_us/open_edx_release_notes/source/koa.rst [hunk 2]
```diff
@@ -4,7 +4,202 @@
...
+
+Mobile Course Experience
+........................
+
+**Native Dates Area**: Previously we used to display a webview on the Dates screen which loaded a course's important dates. This has now been implemented in a native view fully built on mobile (More implementation details of the native dates tab can be found here.
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst]: > (More implementation details of the native dates tab can be found here.

Missing link and closing parenthesis.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 3]
```diff
@@ -4,7 +4,202 @@
...
+
+**Personalized Learning Schedules on Mobile**: As in the desktop experience, our mobile users will also have the ability to shift their courses' due dates!
+The API details for updated course experience can be found in the 'course' section of the full API list here.
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst]: > of the full API list here.

Missing link.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 4]
```diff
@@ -4,7 +4,202 @@
...
+
+
+=========================
+Administrator Experiences
+=========================
+
+Dependency updates
+------------------
+
+These dependencies were upgraded for the `Open edX Koa Installation`_:
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst:112]: I believe this link is missing.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 5]
```diff
@@ -4,7 +4,202 @@
...
+
+
+=====================
+Developer Experiences
+=====================
+
+Pattern Library & Components: Paragon
+-------------------------------------
+
+Beyond the technical improvements noted below, people and processes supporting Paragon have undergone significant changes. Originally conceived as a React component library, Paragon has been expanded to serve as a design system for Open edX. To this end, the new `Design System Documentation` on Confluence is now the source of truth for all things Paragon. https://edx.github.io/paragon will continue to serve its critical function as technical documentation, but will not be expanded to include design documentation in the near term.
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst]: > the new `Design System Documentation`

This is an incorrectly formatted link.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 6]
```diff
@@ -4,7 +4,202 @@
...
+  - `Avatar`__ and `AvatarButton`__, `IconButton`__, `Toast`__
+
+__ https://openedx.atlassian.net/wiki/spaces/BPL/pages/2103083206/Avatar
+__ https://openedx.atlassian.net/wiki/spaces/BPL/pages/2102821019/AvatarButton
+__ https://openedx.atlassian.net/wiki/spaces/BPL/pages/2097250669/Icon+Button
+__ https://edx.github.io/paragon/components/toast
+
+- **Augmented component offering via React Bootstrap**
+
+  - The component offering in Paragon has been expanded by offering pathrough exports to React Bootstrap components (`10.0.0`_, `12.0.0`_). These include: Alert, Badge, Button, ButtonGroup, Card,  Carousel, Dropdown, Figure, Form, InputGroup, Image, Nav, Navbar, Overlay, Popover, ProgressBar, Spinner, Tabs, Tooltip
```
Comment (regisb) [inline en_us/open_edx_release_notes/source/koa.rst]: What does "pathrough" stand for? Is it "passthrough"? (and what is that anyway?)

**Review by regisb — COMMENTED** [review]
Mostly minor issues with links and formatting.

File: en_us/open_edx_release_notes/source/koa.rst [hunk 7]
```diff
@@ -4,7 +4,202 @@
...
+
+**Self-Service Proctoring Provider Selection**: Proctored exam settings have been added to the new front end created to contain future course authoring improvements. A new view, exposed within Studio, has been created to surface proctoring provider selection and settings related to proctoring. This also includes a drop down for selecting a proctored exam provider from a list of available providers.
+
+**IDV experience improvements**: The ID verification (IDV) workflow user experience has been improved by breaking down the steps with clear instructions and examples. Accessibility improvements such as face tracking and voice-over have also been added. This new experience can be viewed in the account micro-frontend.
+
+Course Dates & Milestones
+.........................
+
+**Personalized Learner Schedules** present learners in self-paced courses with suggested assignment due dates based on their enrollment date and the expected course duration. Learners are able to adjust the schedule if they fall behind and miss an assignment. By providing flexibility while also encouraging accountability, Personalized Learner Schedules help keep learners on track to complete their course.
```
Comment (BbrSofiane) [inline en_us/open_edx_release_notes/source/koa.rst:33]: Should a 'How to' guide on setting up personalised learner schedules be part of the release notes? Or at least a link to said guide?

File: en_us/open_edx_release_notes/source/koa.rst (hunk 3 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst]: fixed.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 2 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst]: fixed.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 1 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst]: fixed.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 4 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst:112]: This link is defined in links.rst, it works.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 5 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst]: fixed.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 6 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst]: "pass-through", and I don't know :)

**Comment by nedbat** [comment]
I'm merging what we have, we can tweak from here.

**Comment by nedbat** [comment]
I've cherry-picked this to the Koa branch.

File: en_us/open_edx_release_notes/source/koa.rst (hunk 7 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/koa.rst:33]: I'm happy to link to something.

**Pull request merged**